plugins/woocommerce-admin/client/marketing/data-multichannel/selectors.ts

```diff
@@ -10,3 +10,7 @@ export const getRegisteredChannels = ( state: State ) => {
 export const getRecommendedChannels = ( state: State ) => {
 	return state.recommendedChannels;
 };
+
+export const getCampaigns = ( state: State ) => {
```

Comment:
💅  Missed JSDOCS

Reply:
Done in [`4b87f0a` (#36735)](https://github.com/woocommerce/woocommerce/pull/36735/commits/4b87f0a4ed13d00bd8054687bdab823ba10a6a03).

Reply:
Missed `@param state` declaration